mix.exs

```diff
@@ -45,7 +45,9 @@ defmodule MirraBackend.MixProject do
     [
       arena: [applications: [arena: :permanent]],
       # TODO ArenaLoadTest must deploy only arena
-      arena_load_test: [applications: [arena_load_test: :permanent, arena: :permanent]],
+      arena_load_test: [
+        applications: [arena_load_test: :permanent, arena: :permanent, bot_manager: :pemanent]
+      ],
```

Comment:
Keep in mind that this will conflict with -> https://github.com/lambdaclass/mirra_backend/pull/436/